doc/source/whatsnew/v1.0.0.rst

```diff
@@ -86,7 +86,7 @@ You can use the alias ``"string"`` as well.
    s
 
 The usual string accessor methods work. Where appropriate, the return type
-of the Series or columns of a DataFrame will also have string dtype.
+of the Series or columns of a DataFrame will also have string dtype. (:issue:`29975`)
```

Comment:
move to line 59. @TomAugspurger should add the original issue numbers there as well

Reply:
Got it, thanks!